### PR TITLE
fix(custom clock process): update Custom Clock Processes' doc

### PR DESCRIPTION
**Pull request by Frzk**
Fully reorganized/rewrote the documentation about Custom Clock Processes:
- I calked the page organization on my suggestions for the Scalingo Scheduler (see PR https://github.com/Scalingo/documentation/pull/1915)
- I specifically listed the differences with the Scalingo Scheduler
- I added some precisions about how they work
- I added a 'Costs' section
- I rephrased some explanations
- ...

I hope this will make things a bit clearer for the users.

### CODE REVIEW

File: src/_posts/platform/app/task-scheduling/2000-01-01-custom-clock-processes.md [hunk 1]
```diff
@@ -1,55 +1,114 @@
...
 
-```yaml
-web: bundle exec puma -C config/puma.rb
-clock: bundle exec clockwork clock.rb
-```
+Custom clock processes also work with [Review Apps]({% post_url platform/app/2000-01-01-review-apps.md %}),
+just like the parent application does.
 
-## Start the Task Scheduler
+### Limitations
```
Comment (EtienneM) [inline src/_posts/platform/app/task-scheduling/2000-01-01-custom-clock-processes.md:32]: Just like in the Scalingo scheduler doc page, I would prefer having this section later on the page.

File: src/_posts/platform/app/task-scheduling/2000-01-01-custom-clock-processes.md [hunk 2]
```diff
@@ -1,55 +1,114 @@
...
+  container(s) don't have a limited lifespan.
+
+- Run as many scheduled jobs as you want.
+
+- Have full control over the schedule: if you want a task to repeat every
+  minute, you can.
+
+- Have a very precise schedule, if need be.
+
+### Costs
```
Comment (EtienneM) [inline src/_posts/platform/app/task-scheduling/2000-01-01-custom-clock-processes.md:51]: Maybe in this section we want to insist on the fact that these containers run 24/7 as opposed to Scalingo scheduler one-off containers that are only billed for the time the tasks run.

File: src/_posts/platform/app/task-scheduling/2000-01-01-custom-clock-processes.md [hunk 3]
```diff
@@ -127,37 +188,46 @@ The file which implements the cron-like process is defined in `cron.php`:
...
+The code is available in our [sample-node-express](https://github.com/Scalingo/sample-node-express)
+repository.
 
-#### Example
+This example leverages the `node-cron` package to:
+- run `job1` every 2 minutes
+- run `job2` every minute
 
-The following example uses the `node-cron` package.
+Both jobs just log a message.
```
Comment (EtienneM) [inline src/_posts/platform/app/task-scheduling/2000-01-01-custom-clock-processes.md:222]: best practice: we usually don't use words such as "just", "simply", etc in documentation pages

```suggestion
Both jobs log a message.
```

File: src/_posts/platform/app/task-scheduling/2000-01-01-custom-clock-processes.md [hunk 4]
```diff
@@ -185,7 +255,30 @@ var job2 = new cron.CronJob({
...
+
+Moreover, since Scalingo expects a `web` process type to be defined (either by
+you in the `Procfile`, or automatically by a buildpack), there is most
+probably a `web` process type that would start with your application. **If you
+don't need it**, you can disable it by scaling the `web` process type to zero
+before or after your deployment:
+
+```bash
+scalingo --app my-app scale web:0
+```
```
Comment (EtienneM) [inline src/_posts/platform/app/task-scheduling/2000-01-01-custom-clock-processes.md:275]: suggestion: do you want to add here a link to this page (https://doc.scalingo.com/platform/app/web-less-app)?

File: src/_posts/platform/app/task-scheduling/2000-01-01-custom-clock-processes.md (hunk 2 above)
Comment (Frzk) [reply src/_posts/platform/app/task-scheduling/2000-01-01-custom-clock-processes.md:51]: Fixed in https://github.com/Scalingo/documentation/pull/1935/commits/9f1ebb5dcd891c940c5966552fd38b5f161b0e5e

File: src/_posts/platform/app/task-scheduling/2000-01-01-custom-clock-processes.md (hunk 1 above)
Comment (Frzk) [reply src/_posts/platform/app/task-scheduling/2000-01-01-custom-clock-processes.md:32]: See https://github.com/Scalingo/documentation/pull/1915#discussion_r1097233897

File: src/_posts/platform/app/task-scheduling/2000-01-01-custom-clock-processes.md [hunk 5]
```diff
@@ -1,55 +1,118 @@
...
+modified_at: 2023-02-01 17:00:00
 tags: task-scheduling
 index: 2
 ---
 
-With custom clock processes, you have the ability to specify custom schedule. You will also have other benefits like
-environment parity between development and production, or time precision in your task scheduling.
+Scalingo provides a feature called *Custom Clock Processes* to help you run
+tasks on a regular basis. Unlike jobs setup through the [Scalingo Scheduler]({% post_url platform/app/task-scheduling/2000-01-01-scalingo-scheduler.md %}),
+custom clock processes do not suffer from many limitations.
```
Comment (Soulou) [inline src/_posts/platform/app/task-scheduling/2000-01-01-custom-clock-processes.md:11]:
```suggestion
custom clock processes do not suffer from the same limitations.
```

File: src/_posts/platform/app/task-scheduling/2000-01-01-custom-clock-processes.md [hunk 6]
```diff
@@ -1,55 +1,118 @@
...
 nav: Custom Clock Processes
-modified_at: 2021-10-19 10:00:00
+modified_at: 2023-02-01 17:00:00
 tags: task-scheduling
 index: 2
 ---
 
-With custom clock processes, you have the ability to specify custom schedule. You will also have other benefits like
-environment parity between development and production, or time precision in your task scheduling.
+Scalingo provides a feature called *Custom Clock Processes* to help you run
```
Comment (Soulou) [inline src/_posts/platform/app/task-scheduling/2000-01-01-custom-clock-processes.md:9]: We don't really have such "feature", it's not a name which can be found elsewhere from here, I'm not convinced about highlighting it like here.

File: src/_posts/platform/app/task-scheduling/2000-01-01-custom-clock-processes.md [hunk 7]
```diff
@@ -1,55 +1,118 @@
...
+- Run as many scheduled jobs as you want.
+
+- Have full control over the schedule: if you want a task to repeat every
+  minute, you can.
+
+- Have a very precise schedule, if need be.
+
+### Costs
+
+The feature itself it completely free, but the container in which the
```
Comment (Soulou) [inline src/_posts/platform/app/task-scheduling/2000-01-01-custom-clock-processes.md:53]: I don't really like calling it "the feature", it's not present in the dashboard nowhere, it's not "free". Or I would say "defining a new type of container for your custom clock is free, but the container [...]"

File: src/_posts/platform/app/task-scheduling/2000-01-01-custom-clock-processes.md (hunk 5 above)
Comment (Frzk) [reply src/_posts/platform/app/task-scheduling/2000-01-01-custom-clock-processes.md]: Fixed in https://github.com/Scalingo/documentation/pull/1935/commits/a7a03d02828f4da629fa74557a6f3ed9c866f9c6

File: src/_posts/platform/app/task-scheduling/2000-01-01-custom-clock-processes.md (hunk 6 above)
Comment (Frzk) [reply src/_posts/platform/app/task-scheduling/2000-01-01-custom-clock-processes.md]: I updated the wording in https://github.com/Scalingo/documentation/pull/1935/commits/a7a03d02828f4da629fa74557a6f3ed9c866f9c6
Please tell me if it's better?

File: src/_posts/platform/app/task-scheduling/2000-01-01-custom-clock-processes.md (hunk 7 above)
Comment (Frzk) [reply src/_posts/platform/app/task-scheduling/2000-01-01-custom-clock-processes.md]: I updated the paragraph in https://github.com/Scalingo/documentation/pull/1935/commits/a7a03d02828f4da629fa74557a6f3ed9c866f9c6
Please tell me if it's better?

**Review by Soulou — APPROVED** [review]

**Pull request merged**